core/src/main/java/nu/yona/server/messaging/service/MessageService.java

```diff
@@ -248,6 +253,19 @@ public void sendMessage(Message message, MessageDestination destinationEntity)
 	}
 
 	@Transactional
+	public void broadcastSystemAdminMessage(String messageText)
+	{
+		userAnonymizedService.getAllUsersAnonymized()
+				.forEach(userAnonymized -> sendSystemAdminMessage(messageText, userAnonymized));
```

Comment:
No, because if it than fails in the middle, the other users won't get a message. The batch framework keeps track of where it left it and can resume from there.